kolibri/core/assets/src/views/AttemptLogItem.vue

```diff
@@ -59,7 +59,7 @@
       v-if="attemptLog.missing_resource"
       class="coach-content-label"
       icon="warning"
-      :color="$themePalette.yellow.v_1100"
+      :color="$themePalette.yellow.v_500"
```

Comment:
Just a note: v_1100 values map to v_600, so we will need to fix the current changes that changed v_1100 to v_500